metrics/pcp/pcp.go

```diff
@@ -0,0 +1,129 @@
+package pcp
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/performancecopilot/speed"
+)
```

Comment:
Please group imports according to the [canonical style](https://github.com/golang/go/wiki/CodeReviewComments#imports), that is,

```go
import (
    // stdlib

    // third-party, not Go kit

    // Go kit 
)
```